ui/Main.elm

```diff
@@ -21,8 +22,13 @@ main =
 update : Msg -> Model -> ( Model, Cmd Msg )
 update msg ({ filterValues } as model) =
     case msg of
-        BuildRecordsFetched (Ok buildRecordList) ->
-            updateModelWithFilters ({ model | builds = buildRecordList, loading = False })
+        BuildRecordsFetched (Ok buildsPager) ->
+            updateModelWithFilters
+                { model
+                    | buildsPager = buildsPager
+                    , filteredBuilds = buildsPager.objects
```

Comment:
Very well spotted, thanks!
And pulling that string, it came with a bunch of other dead code <3